components/TextControls/tests/snapshot/supplemental/MDCBaseTextFieldTestsSnapshotTestHelpers.m

```diff
@@ -154,6 +157,31 @@ + (void)configureTextFieldWithColoredAssistiveLabelTextWhileDisabled:(MDCBaseTex
   textField.enabled = NO;
 }
 
++ (void)configureTextFieldWithScaledFontsAndAXXXLargeContentSize:(MDCBaseTextField *)textField {
+  [MDCTextControlSnapshotTestHelpers
+      applyContentSizeCategory:UIContentSizeCategoryAccessibilityExtraExtraExtraLarge];
+
+  textField.text = @"text";
+  textField.label.text = @"label text";
+  textField.leadingAssistiveLabel.text = @"leading assistive label text";
+  textField.adjustsFontForContentSizeCategory = YES;
+
+  if (@available(iOS 11.0, *)) {
+    UIFontMetrics *bodyMetrics = [UIFontMetrics metricsForTextStyle:UIFontTextStyleBody];
+    UIFontMetrics *caption2Metrics = [UIFontMetrics metricsForTextStyle:UIFontTextStyleCaption2];
```

Comment:
Tip: In the future I would avoid using the `.caption` metrics. They tend not to scale as much as metrics like Body.  For some smaller base point sizes, it can make it hard to determine if scaling is taking place for the smaller content size categories.